persistenceagent/rockcraft.yaml

```diff
@@ -0,0 +1,61 @@
+name: kfp-persistence
+summary: Reusable end-to-end ML workflows built using the Kubeflow Pipelines SDK
+description: |
+  This component serves as the backend persistence agent of Kubeflow pipelines.
+version: 2.0.0-ibeta.1_22.04_1 # version format: <KF-upstream-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:22.04
+services:
+  persistenceagent:
+    override: replace
+    summary: "persistenceagent service"
+    startup: enabled
+    user: appuser
+    environment:
+      NAMESPACE: ""
+      TTL_SECONDS_AFTER_WORKFLOW_FINISH: 86400
+      NUM_WORKERS: 2
+    command: bash -c 'persistence_agent --logtostderr=true --namespace=$NAMESPACE --ttlSecondsAfterWorkflowFinish=$TTL_SECONDS_AFTER_WORKFLOW_FINISH --numWorker $NUM_WORKERS'
+platforms:
+  amd64:
+
+parts:
+  persistenceagent:
+    plugin: go
+    source: https://github.com/kubeflow/pipelines
+    source-type: git
+    source-tag: 2.0.0-beta.1 # upstream branch
```

Comment:
Are we sure we want beta tag ?

Reply:
Shoud be `[2.0.0-alpha.7](https://github.com/kubeflow/pipelines/commit/c9fe1166e8af114637d6e6185ced1d3f6bf4b9eb)`